pkg/multicluster/register/internal/cluster_rbac_binder.go

```diff
@@ -0,0 +1,143 @@
+package internal
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rotisserie/eris"
+	k8s_core_v1 "github.com/solo-io/skv2/pkg/generated/kubernetes/core/v1"
+	rbac_v1 "github.com/solo-io/skv2/pkg/generated/kubernetes/rbac.authorization.k8s.io/v1"
+	k8s_rbac_types "k8s.io/api/rbac/v1"
+	k8s_meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	EmptyRolesListError        = eris.New("Empty Roles list found, must specify at least one role to bind to.")
+	EmptyClusterRolesListError = eris.New("Empty ClusterRoles list found, must specify at least one role to bind to.")
+)
+
```

Comment:
it's still not clear to me what the code in this file is used for. it would be helpful to comment what the `clusterAuthorization` does and where it is called / used

Reply:
`clusterAuthorization` is a leftover from a previous iteration. This should be named `clusterRbacBinder`. It is the implementation of the `ClusterRBACBinder` interface.